appservice/src/remoteDebug/startRemoteDebug.ts

```diff
@@ -124,3 +110,38 @@ async function getDebugConfiguration(context: IActionContext, language: RemoteDe
 
     return config;
 }
+
+function getPythonDebugConfiguration(sessionId: string, portNumber: number, host: string): vscode.DebugConfiguration {

```

Comment:
Node.js remote debugging worked for me when I tested your changes, but I get this for Python:
![Screen Shot 2021-08-10 at 11 03 09 AM](https://user-images.githubusercontent.com/11282622/128912854-3c9eba5e-bb85-4024-bfc5-91a84db8478b.png)

Any ideas?

Reply:
It needs to be an ANT-95 image, I've had success with West Central US. Not all regions have been updated yet